src/Alert.php

```diff
@@ -4,269 +4,379 @@
+     *
+     * @return self A new instance of the current class with the specified class value.
+     *
+     * @link https://html.spec.whatwg.org/#classes
      */
-    public function body(string|Stringable $value): self
+    public function addClass(string $value): self
     {
         $new = clone $this;
-        $new->body = $value;
+        $new->addClasses[] = $value;
 
         return $new;
     }
 
     /**
-     * The header content in alert component
+     * Sets the HTML attributes for the alert component.
+     *
+     * @param array $values Attribute values indexed by attribute names.
+     *
+     * @return self A new instance of the current class with the specified attributes.
+     *
+     * @see {\Yiisoft\Html\Html::renderTagAttributes()} for details on how attributes are being rendered.
      */
-    public function header(?string $header): self
+    public function attributes(array $values): self
     {
         $new = clone $this;
-        $new->header = $header;
+        $new->attributes = $values;
 
         return $new;
     }
 
     /**
-     * The HTML attributes for the widget header tag. The following special options are recognized.
+     * Sets the body content of the alert component.
+     *
+     * @param string|Stringable $value The body content of the alert component.
+     * @param bool $encode Whether the body value should be HTML-encoded. Use this when rendering user-generated content
+     * to prevent XSS attacks.
      *
-     * {@see Html::renderTagAttributes()} for details on how attributes are being rendered.
+     * @return self A new instance of the current class with the specified body content.
      */
-    public function headerOptions(array $options): self
+    public function body(string|Stringable $value, bool $encode = true): self
     {
+        if ($encode === true) {
+            $value = Html::encode($value);
+        }
+
         $new = clone $this;
-        $new->headerOptions = $options;
+        $new->body = $value;
 
         return $new;
     }
 
     /**
-     * Set tag name for header
+     * Makes the alert dismissible by adding a close button.
      *
-     * @psalm-param non-empty-string $tag
+     * @return self A new instance of the current class with dismissible added.
      */
-    public function headerTag(string $tag): self
+    public function dismissable(): self
     {
         $new = clone $this;
-        $new->headerTag = $tag;
+        $new->dismissable = true;
+        $new->addClasses[] = 'alert-dismissible';
 
         return $new;
     }
 
     /**
-     * The HTML attributes for the widget container tag. The following special options are recognized.
+     * Adds a fade effect to the alert.
      *
-     * {@see Html::renderTagAttributes()} for details on how attributes are being rendered.
+     * @return self A new instance of the current class with the fade effect added.
      */
-    public function options(array $value): self
+    public function fade(): self
```

Comment:
Add bool parameter, that enable/disable fade.

Reply:
done.